src/components/TransactionItem/TransactionItemCell/index.tsx

```diff
@@ -69,8 +69,16 @@ const handleAddressText = (address: string) => {
   return adaptPCEllipsis(address, 5, 80)
 }
 
+const isDaoDepositCell = (cellType: string) => {
+  return cellType === 'nervos_dao_deposit'
+}
+
+const isDaoWithdrawCell = (cellType: string) => {
+  return cellType === 'nervos_dao_withdrawing'
+}
+
 const isDaoCell = (cellType: string) => {
-  return cellType === 'nervos_dao_deposit' || cellType === 'nervos_dao_withdrawing'
+  return isDaoDepositCell(cellType) || isDaoWithdrawCell(cellType)
```

Comment:
Why use two functions to check deposit and withdraw cell, and any other places use them?

Reply:
because at the moment, we don't have withdraw view. after we have withdraw view we can delete `isDaoCell`